app/liquid/liquid_renderer.rb

```diff
@@ -73,6 +73,7 @@ def member_data
   def show_direct_debit?
     recurring_default = @url_params[:recurring_default] || isolate_from_plugin_data(:recurring_default).first
     DirectDebitDecider.decide([@location.try(:country_code), @member.try(:country)], recurring_default)
+    true
```

Comment:
as useful as this is for click testing, I don't want it merged into `going-cardless`